src/main/webapp/filterStyle.css

```diff
@@ -65,8 +65,23 @@
   width: 300px;
 }
 
-.filter {
-  background: #69d1dd;
+.unclicked-filter {
```

Comment:
I think you should keep the `.filter` rule and class and keep the set of attributes for `.unclicked-filter` and `.clicked-filter` minimal, just overriding the background color (could even make the unclicked background part of `.filter` and just override it for `.clicked-filter`*)

* This may get in to [CSS specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity).